rackcity/views/it_instance_views.py

```diff
@@ -335,21 +335,44 @@ def instance_bulk_upload(request):
                     {"failure_message": failure_message},
                     status=HTTPStatus.BAD_REQUEST
                 )
-        # should this be whitespace/caps insenstive? Maybe later
-        if instance_data['hostname'] in hostnames_in_import:
+
+        # Check that all hostnames in file are case insensitive unique
+        instance_data_hostname_lower = instance_data['hostname'].lower()
+        if instance_data_hostname_lower in hostnames_in_import:
             failure_message = "Hostname must be unique, but '" + \
-                instance_data['hostname'] + \
+                instance_data_hostname_lower + \
                 "' appears more than once in import. "
             return JsonResponse(
                 {"failure_message": failure_message},
                 status=HTTPStatus.BAD_REQUEST
             )
         else:
-            hostnames_in_import.add(instance_data['hostname'])
-        try:
-            existing_instance = ITInstance.objects.get(
-                hostname=instance_data['hostname'])
-        except ObjectDoesNotExist:
+            hostnames_in_import.add(instance_data_hostname_lower)
+
+        existing_instance_filtered = ITInstance.objects.filter(
+            hostname__iexact=instance_data['hostname'])
+        if len(existing_instance_filtered) == 1:
```

Comment:
checks if any version of instance_data['hostname'] (i.e. any variation of `server`, `Server`, `seRvEr` etc) exists in the database